pkg/graphql/schema.go

```diff
@@ -67,11 +67,17 @@ const schema string = `
         # Transaction is the transaction that generated this log entry.
         transaction: Transaction
 
-        # CID for the Receipt IPLD block this Log exists in.
+        # CID for the Receipt IPLD block of leaf node.
         cid: String!
 
-        # IPLD block data for the Receipt this Log exists in.
+        # ReceiptCID for the Receipt IPLD block this Log exists in.
+        receiptCID: String!
+
+        # IPLD block data for the Log Leaf node.
         ipldBlock: Bytes!
+
+        # Status of the Receipt IPLD block this Log exists in.
+        status: Int!
```

Comment:
What will this return for pre-byzantium fork records? 

https://ethereum.stackexchange.com/questions/28889/what-is-the-exact-meaning-of-a-transactions-new-receipt-status-field

Reply:
```
// Assign receipt status or post state.
if len(receipt.PostState) > 0 {
    fields["root"] = hexutil.Bytes(receipt.PostState)
} else {
    fields["status"] = hexutil.Uint(receipt.Status)
}
```
@ashwinphatak 
As per my understanding pre-byzantium fork, we used to return `PostState` which is the root and if it is not present then we return the `status`.
For GQL we need to change the type of status to `Bytes!` so it can accommodate both. Or should I add two separate fields `root` and `status`

Reply:
> `PostState` which is the root

Root of what? Is there some notion of success/failure by inspecting the `PostState`?

Reply:
Intermediate state root when the transaction is being applied if the transaction fails 
Source: https://github.com/ethereum/EIPs/issues/98#issue-151094725
But then they removed the `root` field and add `status` in [EIP-658](https://eips.ethereum.org/EIPS/eip-658) this field was replaced by `status`

Reply:
As per https://eips.ethereum.org/EIPS/eip-658, "With the introduction of the REVERT opcode in EIP140, it is no longer possible for users to assume that a transaction failed iff it consumed all gas". 

So, we may be able to return 1 or 0 based on whether the tx consumed all gas for pre-byzantium receipts?

@i-norden what do you think?

Reply:
I don't think we can assume whether or not a tx failed based on how much of the gas limit was used.

If a tx is reverted early on in its execution, due to some programmatic failure of the OPCODE execution itself, then it can be reverted early on and the amount of gas used can be much lower than the limit allocated. But, if a tx reverts *because* there is not enough gas allocated, then it will only revert after the miner has executed all of the OPCODEs it can afford to before bumping up against the limit. In this case the reverted tx may use all or most of the gas limit.

Another edge cases that make this approach infeasible is, for example, if someone vastly overestimated the gas limit needed for a tx. The tx is properly executed but done so far below the allocated limit, so using this heuristic the tx would look like it failed when it didn't.

Reply:
Oh I misread, you meant for pre-byzantium blocks.

Reply:
Theoretically it is possible that a tx uses up the entire gas limit during normal, successful execution- no? And a failed tx But the statement in that EIP makes it sound like it is a safe assumption.

Reply:
Done